easyInterface/Diffraction/Interface.py

```diff
@@ -547,7 +581,7 @@ def setExperiment(self, experiment: Experiment) -> NoReturn:
         if isinstance(experiment, Experiment):
             new_phase_name = experiment['name']
             if new_phase_name in self.project_dict['experiments'].keys():
-                k, v = self.project_dict.getItemByPath(['experiments', new_phase_name]).dictComparison(experiment)
+                k, v, t = self.project_dict.getItemByPath(['experiments', new_phase_name]).dictComparison(experiment)
```

Comment:
Should we use a less cryptic names, such as `keys`, `values`? And what is `t`? It seems that `t` is not used.

Reply:
Modified